examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -222,6 +222,25 @@ CHIP_ERROR PairingCommand::AddWiFiNetwork()
     return mCluster.AddWiFiNetwork(successCallback, failureCallback, mSSID, mPassword, kBreadcrumb, kTimeoutMs);
 }
 
+chip::ByteSpan PairingCommand::GetThreadNetworkId()
+{
+    // For Thread devices the networkId is the extendedPanId and it is
+    // part of the dataset defined by OpenThread
+
+    Thread::OperationalDataset dataset;
+    CHIP_ERROR err = dataset.Init(mOperationalDataset);
+    if (err == CHIP_NO_ERROR)
```

Comment:
nit: I believe if we invert the conditions, we could reduce nesting:

```cpp
Thread::OperationalDataset dataset;

if (dataset.Init(mOperationalDataSet) != CHIP_NO_ERROR) {
  return ByteSpan();
}

if (dataset.GetExtendedPanId(mExtendedPanId) != CHIP_NO_ERROR) {
  return ByteSpan();
}

return ByteSpan(mExtendedPanId);
```

Reply:
Applied change.